config/events.yaml

```
@@ -19,3 +19,9 @@ sylius_ui:
                 content:
                     template: '@SyliusPriceHistoryPlugin/Admin/Product/Show/_pricing.html.twig'
                     priority: 10
+                    
+        sylius.shop.product.show.right_sidebar:
+            blocks:
+                price_widget:
+                    template: "@SyliusPriceHistoryPlugin/Shop/Product/Show/_priceWidget.html.twig"
+                    priority: 30
```

```suggestion
        sylius.shop.product.show.right_sidebar:
            blocks:
                price_widget:
                    template: "@SyliusPriceHistoryPlugin/Shop/Product/Show/_priceWidget.html.twig"
```
You don't have to include priority if it's the same as in the base block